data_generators/user_generators.py

```diff
@@ -47,11 +48,11 @@ def __init__(self):
         self.formerly_incarcerated = percent_true(80)
         self.underrepresented_group = percent_true(70)
         self.low_income = percent_true(70)
-        self.list_convictions = sample(convictions, k=randint(1, 3))
+        self.list_convictions = sample(convictions, k=randint(1, 3))[0]
```

Comment:
Do we have a way of accounting for more than one conviction?

Reply:
Updated list_convictions to account for more than one conviction, while keep the attribute a string